test/e2e/framework/util.go

```diff
@@ -1327,6 +1328,8 @@ func ExpectNoError(err error, explain ...interface{}) {
 func ExpectNoErrorWithOffset(offset int, err error, explain ...interface{}) {
 	if err != nil {
 		e2elog.Logf("Unexpected error occurred: %v", err)
+		// Dump the stacktrace while an error occurs
+		debug.PrintStack()
 	}
```

Comment:
Although `panic` could print the stacktrace but it will exit the program. So using `debug.PrintStack()` instead.